eogrow/pipelines/download_batch.py

```diff
@@ -117,6 +117,13 @@ def __init__(self, *args: Any, **kwargs: Any):
 
         self.batch_client = SentinelHubBatch(config=self.sh_config)
 
+        self.area_manager = cast(BatchAreaManager, self.area_manager)
```

Comment:
casts can be avoided with the same trick as in the eopatch manager (by annotation)

Reply:
You are right. Fixed.